CHANGELOG.rst

```diff
@@ -6,16 +6,52 @@
 New features
 ~~~~~~~~~~~~
 
-- Logged-in users without developer access can launch interactive sessions from a project.
+⭐️ Environments: logged-in users without developer access can launch interactive sessions from a project.
 
-- Interactive sessions can be enabled for logged-out users. Please see the
-  `documentation <https://renku.readthedocs.io/en/latest/admin/index.html#enabling-notebooks-for-anonymous-users>`__ for details.
+⭐️ Environments: interactive sessions can be enabled for logged-out users. Please see the
+  `documentation <https://renku.readthedocs.io/en/latest/admin/anonymous-sessions.html>`__ for details.
+
+Notable improvements
+~~~~~~~~~~~~~~~~~~~~
+
+🚄 Datasets: dataset creation and import unified in the UI
+
+Bug fixes
+~~~~~~~~~
+
+* Datasets now show file listing with folder hierarchy
+* Datasets: avoid recursive addition of data directory in Renku CLI
```

Comment:
I would suggest "of data directories" or "of the data directory". Not sure which version is actually the correct one for this change.